apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -359,6 +372,26 @@ defmodule AndiWeb.EditLiveView do
      )}
   end
 
+  def handle_info({:remove_data_dictionary_field_succeeded, deleted_field_parent_id, deleted_field_index}, socket) do
+    new_selected_field =
```

Comment:
You figured it out?

Reply:
No just punting, I wrote another card to look at that. In the meantime I am trying to reproduce enough to maybe submit a bug report